magento/pwa-studio
# Increase useApp() talon test coverage

## Description

Increase test coverage for the `useApp()` talon.

## Related Issue

Closes PWA-994

## Acceptance 

Any developer

### Verification Stakeholders

Any developer

### Specification



### Verification Steps

1. Run `yarn test --collectCoverageFrom=**/useApp.js packages/peregrine/lib/talons/App/`
1. Verify tests succeed
1. Verify acceptable test coverage

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
